db/active_replicator_checkpointer.go

```diff
@@ -50,6 +51,8 @@ type Checkpointer struct {
 	lastLocalCheckpointRevID string
 	// lastCheckpointSeq is the last checkpointed sequence
 	lastCheckpointSeq SequenceID
+	// collectionIdx is the replication index of the collection we're checkpointing for
```

Comment:
since collection idx is used in a bunch of different ways, call out that this is the blip collection idx from `GetCollections` sync message.